docs/docs/aleph_zero/storage.md

```diff
@@ -6,7 +6,7 @@ slug: /aleph_zero/storage
 
 This section provides an in-depth exploration of key data structures integral to the Invariant protocol's storage mechanism. These structs are specifically crafted to facilitate the sharing of the state of the exchange within the CLAMM model. These data structures play a pivotal role in maintaining and organizing information related to the exchange, ensuring efficient and organized handling of data.
 
-## Contract state
```

Comment:
the "Contract state" was correct. IMO, a term like 'GlobalState' would be more suitable in the code